pubsub/google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -68,6 +61,12 @@ def __init__(self, batch_settings=(), batch_class=thread.Batch, **kwargs):
                 target=os.environ.get('PUBSUB_EMULATOR_HOST'),
             )
 
+        if 'batch_class' in kwargs:
```

Comment:
Just drop this, we're before GA so we're allowed to remove this.

Reply:
Removing, although the purpose was to give a reasonable exception to anyone happening to use this; without it, it will be passed as kwargs to another constructor later, which will presumably blow up.